deploy/common/k8sbased.sh

```diff
@@ -256,6 +259,10 @@ function launch_central {
 
     echo
     if [[ -n "${TRUSTED_CA_FILE}" ]]; then
+        if [[ "$namespace" != "stackrox" ]]; then
+          echo "CA setup using '$TRUSTED_CA_FILE' is not support for custom namespace '$namespace'." >&2
```

Comment:
qq here without me having any context on the CA setup: why is that and what should someone seeing this message do differently?

Reply:
I am also lacking context here. But after a quick look I had the impression that this might involve more work to be adjusted to really support custom namespaces. Hence, I tried to keep this out of scope for this PR, trying not to break anything here.

Given that I am implementing this custom-namespace support throughout the tooling it is probably safe to say that I am -- at least for now -- becoming the primary or only user of this feature. I need it for proper scanner v4 testing. Therefore, for the time being it is expected that nobody else will hit this code path, because everything within our tooling is still defaulting to the `stackrox` namespace. If somebody needs this kind of CA setup *and* custom namespaces, then this lack of support must of course be investigated and addresses properly.